tests/codemods/test_flask_json_response_type.py

```diff
@@ -0,0 +1,238 @@
+from core_codemods.flask_json_response_type import FlaskJsonResponseType
+from tests.codemods.base_codemod_test import BaseCodemodTest
+from textwrap import dedent
+
+
+class TestFlaskJsonResponseType(BaseCodemodTest):
```

Comment:
What about a test case where a variety of different headers are already set? We need to make sure the only modified header is `Content-Type`.